ruuvitag_sensor/data_formats.py

```diff
@@ -1,3 +1,22 @@
+import os
+
+
+def get_raw_hci(raw, data_format):
+    # pylint: disable=unused-argument
+    return raw
+
+
+def get_raw_bleson(raw, data_format):
+    # Bleson drops FF from the raw data and it is required for DF 3 and 5
+    # TODO: Move convert_data to adaptor specific code
+    if data_format in (2, 4):
+        return raw
+    return 'FF' + raw
+
+
+get_raw = get_raw_bleson if os.environ.get('RUUVI_BLE_ADAPTER') == 'Bleson' else get_raw_hci
```

Comment:
Did you consider this approach from one of my old branches:
```
        #Python duck typing
        try:
            string_data = raw.hex()
            if string_data[0:2] != 'FF':
                string_data = 'FF' + string_data

        except (AttributeError):
            string_data = raw
```

Feels a bit wrong do do env checks here.

Reply:
Yes, I hate using env checks all around the code base. In my opinion env checks should only be done in single place and all functionality should be configured/decided there.

I considered that duck typing, but I don't think that is a good option as when not using Bleson there is lots of unnecessary exceptions. Also using exceptions is not a good way to control the program flow, although in here it is the simplest solution. Better to decide the parsing function to use once.

I changed the functionality a little and use patching. Although don't like to override methods like this, but I think for now this is the best solution. Don't want to make too much refactoring in this commit, so with this I don't have to make changes to multiple places.